### DISCUSSION TITLE
Keep Original Name always on?

### DISCUSSION THREAD

**Discussion by thehijacker** [diced/zipline#570] [Questions]
Hello all,

Sorry but I just can not find it. Using docker to start zipline and it works great together with ShareX. What I wish to have always turned on is "Keep Original Name" option. Now I need to click Options and select it before uploading files manually. How can I enable this permanently?

Thank you.

**Comment by TheAnachronism** [comment]
I've just found the same thing and hope this can be possible.
Preferred defaults would be, Random Name & Keep Original for download.

Hope the second one gets added as a system default option at some point as well.

**Reply by Goeste** [reply]
@thehijacker, I'd even extend it, the option you mention is not working for me, I always get random filenames...

**Comment by matejdro** [comment]
I guess a better way would be to simply remember all Upload options? Maybe saved in a cookie or something.

**Reply by diced** [reply]
This is done in v4 :+1:
